lua/default-config.lua

```diff
@@ -811,8 +730,6 @@ lvim.lang = {
           DATA_PATH .. "/lspinstall/php/node_modules/.bin/intelephense",
           "--stdio",
         },
-        on_attach = common_on_attach,
-        on_init = common_on_init,
         filetypes = { "php", "phtml" },
```

Comment:
Hopefully `common_capabilities` is missing here only due to a copy-paste mistake, and not some weird requirement. The same goes for Kotlin.

Reply:
yeah i saw it, but it seems to have been a copy pasta mistake